frontend/src/editor/commands/removeBorderNumbers.ts

```diff
@@ -0,0 +1,114 @@
+import { CommandProps } from "@tiptap/core"
+import { Node as ProseMirrorNode, NodeType } from "prosemirror-model"
+import { Transaction, TextSelection } from "prosemirror-state"
+import { nextTick } from "vue"
+import BorderNumberService from "@/services/borderNumberService"
+
+/**
+ * Main command handler to remove borderNumber nodes.
+ */
+function removeBorderNumbers(
+  { state, dispatch }: CommandProps,
+  isFeatureEnabled: boolean,
+): boolean {
+  const { selection, doc, tr, schema } = state
+  const { from, to } = selection
+  const borderNumberNodeType: NodeType = schema.nodes.borderNumber
+
+  const borderNumberPositions = findBorderNumberPositions(
+    doc,
+    from,
+    to,
+    borderNumberNodeType,
+  )
+
+  const { modified, borderNumberSizes } = processBorderNumbers(
+    tr,
+    doc,
+    borderNumberPositions,
+  )
+
+  void nextTick().then(() => {
+    if (isFeatureEnabled) {
+      BorderNumberService.makeBorderNumbersSequential()
+    }
+  })
+
+  if (modified && dispatch) {
+    const totalSizeOfRemovedNodes = borderNumberSizes.reduce(
+      (acc, size) => acc + size,
+      0,
+    )
+
+    const selection = TextSelection.create(
+      tr.doc,
+      Math.max(1, from - totalSizeOfRemovedNodes),
```

Comment:
Warum müssen wir hier alle removednodes von `from` abziehen?
```
[ 1 ] First Paragraph <selection start> Text
[ 2 ] Second Paragraph Text
[ 3 ] Third Paragraph <selection end> Text
```
-> If I know press removeBorderNumbers then all three border numbers are removed. However, only the first removed border number will change the position of the selection start / `from` no?